src/rpm/compressor.rs

```diff
@@ -7,6 +7,22 @@ pub enum Compressor {
     Zstd(zstd::stream::Encoder<'static, Vec<u8>>),
 }
 
+impl Default for Compressor {
+    fn default() -> Self {
+        Self::None(Default::default())
```

Comment:
I kinda feel like no compression is a bad default?  Gzip is the default for `rpmbuild` and is supported by everything, Zstd is much better and is supported by everything for the past 5-6 years (e.g. EL8)

Reply:
It's primarily for builder use, where that was the default.

Reply:
I get that it was the default, but is it a good default?  (we can address that separately, not asking for it in this PR)

Reply:
I agree with your statement. Parsing is not affected, it's just the identical default for constructing the builder

Reply:
My background is software for managing RPMs, and our users sometimes have several terabytes of RPMs, so this is the kind of thing I think about.

Reply:
Created a discussion here: https://github.com/rpm-rs/rpm/discussions/127